tutorials/scripting/gdscript/warnings/confusable_identifier.rst

```diff
@@ -0,0 +1,31 @@
+CONFUSABLE_IDENTIFIER
+=========================
+
+The warning message is:
+
+.. code-block:: none
+
+    The identifier "my_vаr" has misleading characters and might be confused with something else.
+
+The default warning level for this warning is **Warn**.
+To modify it, set :ref:`Project Settings > Debug > GDScript > Warnings > Confusable Identifier<class_ProjectSettings_property_debug/gdscript/warnings/confusable_identifier>`.
+
+When this warning occurs
+------------------------
+
+Some alphabets such as Cyrillic have characters that look like Latin (such as English, Spanish, etc.) characters, but are actually different.
+
+.. code-block::
+
+    var engine_nаme = "Godot"
+    print(engine_name)
+
+In this code snippet, the ``print`` statement would fail, because ``engine_name`` is actually not defined. The identifier in the ``print`` statement uses the Latin character "a" (U+0061), while the identifier in the variable declaration above uses the Cyrillic character "а" (U+0430).
+
+How to fix this warning
+-----------------------
+
+Avoid using Cyrillic or other alphabets' characters that are visually similar to Latin ones. A good rule of thumb is to prefer the Latin alphabet for program identifiers.
```

Comment:
```suggestion
Avoid using Cyrillic or other scripts' characters that are visually similar to Latin ones. A good rule of thumb is to prefer the Latin alphabet for program identifiers.
```
Not all writing systems are alphabets (granted most would not be confusable with alphabets but still)

Reply:
The term "script" is consistent to how the rest of the engine refers to the concept, but I believe here "alphabet" was used because otherwise it'd be very much confusable with GDScript itself.

Reply:
Admittedly I think here I just used the term "alphabet" because I was thinking of individual characters; I don't remember taking the confusion with GDScript scripts into consideration 😅 It is a really good point, though. I feel like for now we can use "scripts" here since it is more correct, but revisit it if we get the feeling it's confusing people.

Reply:
Weird, I didn't get a notification about your comment until a while after I'd posted mine...

> I don't think anyone will confuse this with GDScript, why would anyone think "Cyrillic" is a GDScript?

That's fair, although I could also see someone parsing the sentence as "Cyrillic characters, or characters from other GDScript scripts", possibly? As I said in my previous comment, though, it's probably okay to just use "scripts" for now and then revisit if people do find it confusing.